recipes/newmat/all/CMakeLists.txt

```
@@ -0,0 +1,96 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(
+  "Newmat"
+  DESCRIPTION
+    "The package is intended for scientists and engineers who need to \
+    manipulate a variety of types of matrices using standard matrix \
+    operations. Emphasis is on the kind of operations needed in statistical \
+    calculations such as least squares, linear equation solve and eigenvalues."
+  HOMEPAGE_URL "http://www.robertnz.net/nm11.htm"
+  LANGUAGES CXX)
+
+option(SETUP_C_SUBSCRIPTS "Activate SETUP_C_SUBSCRIPTS if you want to use traditional C style element access" OFF)
+if (SETUP_C_SUBSCRIPTS)
+   add_compile_definitions(SETUP_C_SUBSCRIPTS)
+endif()
+message("Build with C style element access ${SETUP_C_SUBSCRIPTS}")
+ 
+# -------------
+# target
+# -------------
+set(source_subfolder_ "src")
+set(lib_name "Newmat")
```

```suggestion
set(lib_name ${PROJECT_NAME})
```

The file `nm_gnu` uses `libnewmat.a` as library name.

---

@AndreyMlashkin please, revert it, the library name should be `newmat`, not `Newmat` it's important keeping the same name provided by the upstream:

nm_gnu.mak:16

```
libnewmat.a:   	$(newmat_lobj)
		$(AR) -cr $@ $(newmat_lobj)
		ranlib $@
```